composer.json

```diff
@@ -60,7 +59,8 @@
         "symfony/process": "^2.7 || ^3.0 || ^4.0",
         "symfony/var-dumper": "^5.0",
         "sebastian/diff": "^4.0",
-        "phpspec/prophecy-phpunit": "^2.0"
+        "phpspec/prophecy-phpunit": "^2.0",
+        "dms/phpunit-arraysubset-asserts": "^0.3.0"
```

Comment:
we test the all the phpactor dependencies using this lock file, so easier to make this `^0.2.1`

Reply:
I'm sorry.  Fixed and force pushed https://github.com/phpactor/phpactor/commit/28a3869a1ab94b41b659ac5e7f0015b8f64744cb